lib/features/profiles/screens/wallet_screen.dart

```diff
@@ -167,25 +158,27 @@ class _WalletScreenState extends State<WalletScreen>
                       ),
                     ],
                   ),
-                  Row(
-                    children: [
-                      ProfileSwitchButton(
-                        name: state.activeProfile.firstName,
-                        onClicked: () {
-                          context.pushNamed(Pages.profileSelection.name);
-
-                          AnalyticsHelper.logEvent(
-                            eventName: AmplitudeEvent.profileSwitchPressed,
-                          );
-                        },
-                      ),
-                    ],
-                  ),
-                ],
-              ),
+                ),
+              ]),
             ],
           ),
         ),
+        floatingActionButtonLocation: FloatingActionButtonLocation.startFloat,
+        floatingActionButton: GivtFAButton(
```

Comment:
a bit unclear name (with abbreviation) as for me

Reply:
Updated